Makefile

```diff
@@ -34,8 +34,8 @@ install:
 check: test
 
 test:
-	# to see more add -v -d -s --nologcapture
-	$(wildcard /usr/bin/nosetests-2.*) -c .noserc
+	# to see more add -vv
+	/usr/bin/pytest
```

Comment:
I guess the same goes here - the tests won't work with a make test as they depend on running in a container. So just removing this should help